### ISSUE TITLE
One bug with function ReadLinesOffsetN

### ISSUE THREAD

**Issue by 851235550** [shirou/gopsutil#1201]
https://github.com/shirou/gopsutil/blob/1ec4629e46a7fcf7c25a9acbf487f8cae61c8c00/internal/common/common.go#L119

![image](https://user-images.githubusercontent.com/12993450/146636940-5cdd81cd-6c61-470e-86ae-448472761549.png)

Hi, all
I found a problem when executing `ReadLinesOffsetN` in a docker container with centos7. Is such that:
1. I put a file in my docker container when I was building docker. The `cat` result of this file is as shown in the first red box in the figure above.
2. I execute in the container the method of `ReadLinesOffsetN` found that the return is an empty slice
3. When I manually add a newline to the file, the result of the cat file is the second red box, At this time, I execute the `ReadLinesOffsetN` method again to get the content
4. When I manually delete the line break in step 3, execute the `ReadLinesOffsetN` method again, and find that the content can also be obtained

In summary, is this a known issue?

I think we should add a judgment like this

![image](https://user-images.githubusercontent.com/12993450/146638231-cbcd05c0-1c09-422a-b279-f48b714b2d62.png)
